interfaces/builtin/udisks2.go

```diff
@@ -404,14 +415,14 @@ func (iface *udisks2Interface) StaticInfo() interfaces.StaticInfo {
 }
 
 func (iface *udisks2Interface) DBusConnectedPlug(spec *dbus.Specification, plug *interfaces.ConnectedPlug, slot *interfaces.ConnectedSlot) error {
-	if !release.OnClassic {
+	if !slot.IsImplicit() {
 		spec.AddSnippet(udisks2ConnectedPlugDBus)
```

Comment:
I see that you didn't add this, but shouldn't we do this without any condition?

Reply:
Right. I am not even sure why we have this method at all. Let's keep this comment open until I find an explanation.

Reply:
This function was removed because it did not make sense.